include/z64math.h

```diff
@@ -39,6 +39,11 @@ typedef struct {
     /* 0x8 */ s32 z;
 } Vec3i; // size = 0xC
 
+typedef struct {
+    /* 0x0 */ f32 distance;
+    /* 0x4 */ s16 rotY;
+} VecPolar; // size = 0x8
```

Comment:
Since this is named after the polar coordinate system, I guess it would be better to use `angle` instead of `rotY`